crates/iota-graphql-e2e-tests/coverage/query/address.md

```diff
@@ -0,0 +1,191 @@
+Query: `address`
+
+```
+{
+  address(address: "0x1") {
+    address
+		objects {
+		  edges {
+		    node {
+		      digest
+		      storageRebate
+		      bcs
+		    }
+		  }
+		}
```

Comment:
Could you please pretty format this to attain consistent indentation and format throughout the documents?